ra_aid/webui/static/script.js

```
@@ -0,0 +1,206 @@
+            this.streamOutput.style.display = 'block';
+        } else if (data.type === 'stream_end') {
+            this.streamOutput.style.display = 'none';
+            this.addToHistory(data.request);
+            this.sendButton.disabled = false;
+        } else if (data.type === 'chunk') {
+            this.handleChunk(data.chunk);
+        }
+    }
+
+    handleChunk(chunk) {
+        if (chunk.agent && chunk.agent.messages) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)